text_extensions_for_pandas/io/conll.py

```diff
@@ -138,7 +138,7 @@ def add_line(self, line_num: int, line_elems: List[str]):
 def _parse_conll_file(input_file: str,
                       column_names: List[str],
                       iob_columns: List[bool]) \
-        -> List[List[_SentenceData]]:
+        -> (List[List[_SentenceData]], List[List[int]]):
```

Comment:
Can you please fold these line numbers into the existing `_SentenceData` data structure? Then the return type of this function won't need to be a tuple.